solr/core/src/test/org/apache/solr/handler/RequestHandlerBaseTest.java

```diff
@@ -89,9 +116,16 @@ public void testEach4xxSolrExceptionIncrementsTheClientErrorCount() {
 
     RequestHandlerBase.processErrorMetricsOnException(e, metrics);
 
-    verify(metrics.numErrors).mark();
-    verify(metrics.numClientErrors).mark();
-    verifyNoInteractions(metrics.numServerErrors);
+    verify(mockLongCounter, times(1))
+        .add(eq(1L), argThat(attrs -> "errors".equals(attrs.get(AttributeKey.stringKey("type")))));
```

Comment:
I suspect the idea here in this OTEL API is that you're expected to declare `AttributeKey.stringKey("type")` as a constant somewhere, perhaps TYPE_ATTR